beetsplug/absubmit.py

```diff
@@ -79,7 +82,7 @@ def __init__(self):
             except OSError:
                 raise ui.UserError(
                     'No extractor command found: please install the extractor'
-                    ' binary from https://acousticbrainz.org/download'
+                    ' binary from https://essentia.upf.edu/'
```

Comment:
The download links on the AcousticBrainz site are dead now, so I updated this to the official Essentia project homepage where it's still available.

Reply:
Actually it's only the 64-bit Linux version that's dead, but it's probably still a good idea to reference the official project site instead of the mirrored downloads on the AcousticBrainz site which may not be around for long.